content/articles/2018-04-readxl-1-1-0.Rmd

```diff
@@ -0,0 +1,154 @@
+description: >
+  readxl 1.1.0 is now on CRAN.
+author: Jenny Bryan
+date: 2018-04-17
+categories:
+  - package
+photo:
+  url: https://twitter.com/Thoughtfulnz/status/987900521309614080
+  author: David Hood
+output:
+  blogdown::html_page:
+    css: "/css/article-image.css"
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+
+# ideas for photos before @thoughtfulnz provided his photos!
+
+# https://www.pinterest.com.au/pin/275001120966367679/
+# https://www.flickr.com/photos/mazzapix/5904512406/in/photostream/
+# https://mymodernmet.com/pietro-dangelo-paperclip-sculptures/
+# https://www.themarysue.com/paperclip-art/
+# http://365project.org/cjphoto/365/2017-07-10
+# http://www.cjohnsonphoto.com/2017/07/03/latest-365-project-creatve-paperclip-theme/
+```
+
+We are pleased to announce that [readxl](http://readxl.tidyverse.org) 1.1.0 is now available on CRAN. Learn more about readxl at <http://readxl.tidyverse.org>.
+
+The readxl package makes it easy to get tabular data out of Excel files and into R with code, not mouse clicks. It supports both the legacy `.xls` format and the modern XML-based `.xlsx` format. readxl is expressly designed to be easy to install and use on all operating systems. Therefore it has no external dependencies, such as Java or Perl, which have historically been a source of aggravation with some R packages that read Excel files. 
+
+The easiest way to install the latest version from CRAN is to install the whole tidyverse.
+
+```{r eval = FALSE}
+install.packages("tidyverse")
+```
+
+Alternatively, install just readxl from CRAN:
+
+```{r eval = FALSE}
+install.packages("readxl")
+```
+
+Regardless, you will still need to load readxl explicitly via `library(readxl)`, because it is not a core tidyverse package loaded via `library(tidyverse)`. Do this like so in your scripts:
+
+```{r eval = FALSE}
+library(readxl)
+```
+
+readxl was last released almost exactly one year ago, in April 2017, at version 1.0.0. The associated [blog post](http://readxl.tidyverse.org/articles/releases/readxl-1.0.0.html) summarizes many nifty new features in version 1.0.0. In contrast, version 1.1.0 is considerably less exciting for most users, but includes two important improvements:
+
+  * Security patches in the embedded libxls library
+  * Better support for detecting the format of files, i.e. whether to read as xls or xlsx
+  
+Version 1.1.0 also includes several other small features and bug fixes. For full details on all versions, see the [changelog](http://readxl.tidyverse.org/news/index.html).
+
+Read (or skip!) to the end for a story about how readxl was used in a recent Los Angeles Times piece about homeless arrests. Or, rather, was not used.
+
+## Security fixes for libxls
+
+readxl embeds the library [libxls](https://github.com/evanmiller/libxls) in order to read xls files. In fall 2017, three security vulnerabilities were identified in libxls and were shown to affect readxl:
```

Comment:
I would consider taking out the date here, or if you keep it explain what made it challenging to address the issues until now.